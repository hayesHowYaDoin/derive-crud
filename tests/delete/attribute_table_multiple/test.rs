use derive_crud::Delete;

#[derive(Delete)]
#[crud_table("test_table")]
#[crud_table("test_table2")]
struct Test {
    #[crud_id]
    id: i64,
    name: String,
}

#[tokio::main]
async fn main() {
    let _result = Test::delete(1).await;
}
