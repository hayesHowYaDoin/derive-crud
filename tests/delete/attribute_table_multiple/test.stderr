error: Only one `#[crud_table]` attribute is allowed
 --> tests/delete/attribute_table_multiple/test.rs:5:3
  |
5 | #[crud_table("test_table2")]
  |   ^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0599]: no function or associated item named `delete` found for struct `Test` in the current scope
  --> tests/delete/attribute_table_multiple/test.rs:14:25
   |
6  | struct Test {
   | ----------- function or associated item `delete` not found for this struct
...
14 |     let _result = Test::delete(1).await;
   |                         ^^^^^^ function or associated item not found in `Test`
